source/SIL.AppBuilder.Portal.Frontend/tests/helpers/index.tsx

```diff
@@ -14,7 +13,7 @@ class TestWrapper extends React.Component<any, any> {
     const { initialState, history } = this.props;
     // mount the whole app by default, but allow testing individual
     // components.
-    const ComponentToTest = this.props.component || TodoMVC;
+    const ComponentToTest = this.props.component;
```

Comment:
The root-level component should still be here. (It's not TodoMVC, though, it's 'Application')